compiler/rustc_privacy/src/lib.rs

```diff
@@ -2069,7 +2069,6 @@ impl<'tcx> Visitor<'tcx> for PrivateItemsInPublicInterfacesVisitor<'tcx> {
             // Subitems of trait impls have inherited publicity.
             hir::ItemKind::Impl(ref impl_) => {
                 let impl_vis = ty::Visibility::of_impl(item.def_id, tcx, &Default::default());
-                self.check(item.def_id, impl_vis).generics().predicates();
```

Comment:
This change affects inherent impls too, which contradicts the PR description.

Reply:
Also, could you add a comment saying that generic parameters and where clauses are specifically NOT checked, and it's not an accidental omission, and link to this PR.

Reply:
Fixed with https://github.com/rust-lang/rust/pull/90586/commits/6a5c0d6039e6095edfb302b4902997b76c46ce35!